cves/CVE-2011-1183.yml

```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-269 - Improper Privilege Management
```

Comment:
This should be just the CWE number based on what we went over in class.

Reply:
I may have missed the class when we talked about this. Were we restricted to a subset of the common weaknesses?

Reply:
I changed it to just the number, if that's what you meant.

Reply:
Yup, just the number. We talked about it the Friday it was due.